README.md

```diff
@@ -1,142 +0,0 @@
-# Mainflux
```

Comment:
Why README.md is removed?

Reply:
Huh, good catch @dusanb94 - indeed I do not see it here: https://github.com/darkodraskovic/mainflux/tree/ui_rebase

@darkodraskovic please bring us back README.md in the root folder.

Reply:
Added back.

Reply:
Cool, thanks!